account_comment_template/tests/test_account_move_report.py

```diff
@@ -1,44 +1,64 @@
 # Copyright 2017 Simone Rubino - Agile Business Group
 # Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo.tests.common import TransactionCase
+from odoo.tests.common import Form, TransactionCase
 
 
 class TestAccountInvoiceReport(TransactionCase):
-    at_install = False
-    post_install = True
-
     def setUp(self):
-        super(TestAccountInvoiceReport, self).setUp()
+        super().setUp()
+        self.company = self.env.ref("base.main_company")
         self.base_comment_model = self.env["base.comment.template"]
+        self.move_obj = self.env.ref("account.model_account_move")
         self.before_comment = self._create_comment("before_lines")
         self.after_comment = self._create_comment("after_lines")
         self.partner = self.env["res.partner"].create({"name": "Partner Test"})
-        self.invoice_model = self.env["account.move"]
-        self.invoice = self.invoice_model.create(
+        self.partner.base_comment_template_ids = [
+            (4, self.before_comment.id),
+            (4, self.after_comment.id),
+        ]
+        self.income_account = self.env["account.account"].search(
+            [("user_type_id.name", "=", "Income")], limit=1
```

Comment:
You should also filter by company here. The test user might have more than one company associated and this search will return an account belonging to a different company:
```suggestion
            [("user_type_id.name", "=", "Income"), ("company_id", "=", self.company.id)], limit=1
```